akd/src/append_only_zks.rs

```diff
@@ -29,13 +29,14 @@ use keyed_priority_queue::{Entry, KeyedPriorityQueue};
 
 /// The default azks key
 pub const DEFAULT_AZKS_KEY: u8 = 1u8;
+/// The default location of the azks root
+pub const DEFAULT_AZKS_ROOT: u64 = 0;
```

Comment:
The same value is being committed to the database repeatedly since the root location is determined on the first initialization. But if the flexibility of a potentially dynamic root location is worth it, I can revert this change.